roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -18,11 +18,11 @@
   tags: ngi_reports
 
 - name: Unpack TeXLive
-  shell: "tar -xzf {{ texLive_dest }}/src/{{ texLive_file }} -C {{ texLive_dest }}/src/ --strip-components=1"
+  shell: "tar -xzf {{ texLive_dest }}/src/{{ texLive_file }} -C {{ texLive_dest }}/src/ --strip-components=1 --mode='g+rwxs'"
```

Comment:
Make it a capital `X` instead. That way only directories added will have the executable flag set, and not all text files etc.